database/src/migrations/20230929103700_ignore_project_coordinator.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex';
+
+/**
+ * Makes the Project Coordinator properties on a project nullable, since the app no longer supports
+ * entering or editing project coorindators.
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+    SET search_path = 'biohub';
+
+    ALTER TABLE project
+    ALTER COLUMN coordinator_first_name       DROP NOT NULL,
```

Comment:
Would you mind also updating the comments for the coordinator fields, and just appending something like "Deprecated" to the start. Just in case.

Reply:
Great idea, added.